packages/tasks/src/model-libraries.ts

```diff
@@ -212,6 +212,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/cartesia-ai/cartesia_mlx",
 		snippets: snippets.cartesia_mlx,
 	},
+	"clip-vit-l-scope": {
+		prettyLabel: "CLIP-ViT-L-scope",
+		repoName: "CLIP-ViT-L-scope",
```

Comment:
```suggestion
	"clipscope": {
		prettyLabel: "clipscope",
		repoName: "clipscope",
```

Given the library is https://github.com/Lewington-pitsos/clipscope, I'd recommend to tag the model as "clipscope" instead.